libs/shared/form-fields/codegen.yml

```diff
@@ -0,0 +1,27 @@
+schema:
```

Comment:
IMO: It does not make sense to add a codegen here. This is a shared lib, used by all applications, and it should not have any graphql fields.